app/posts/publish-teacher-training-courses/2021-06-03-placement-schools-and-training-centres.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Placement schools and training centres
+description: Asking providers to give more detailed location information
+date: 2021-06-14
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+## The location problem
+
+Currently, Publish uses a model inherited from UCAS. This allows courses to show 2 types of location:
+
+* the training provider’s contact address
+their training locations – places a candidate
+* can apply to, but not necessarily where they will spend the majority, if any, of their time.
+
+Providers can add up to 37 addresses. Some set themselves up as multiple providers so they can add more locations (such as [West London Teaching School Alliance](https://web.archive.org/web/20210125160621/https://www.find-postgraduate-teacher-training.service.gov.uk/results/filter/provider?l=3&prev_l=none&prev_lat=none&prev_lng=none&prev_loc=none&prev_lq=none&prev_query=none&prev_rad=none&query=West+London+Teaching+School+Alliance)). Others only list one address (this is the case for 50% of SCITTs and 100% of HEIs).
+
+Listed addresses are not classified in any way (they could be administrative addresses, campuses, schools, hubs or areas).
+
+This is the [location problem](/publish-teacher-training-courses/the-location-problem); the information we have is inconsistent, which makes choosing courses based on location impossible for candidates.
+
+## What location data do we need?
+
+We’ve tried to use existing location data on Find, but it has proven problematic:
```

Comment:
I suggest adding a full stop here, there starting a new lead in line with 'We found that:'